scripts/pylint_extensions.py

```diff
@@ -1385,7 +1385,11 @@ def process_module(self, node):
 
 
 class SpaceBelowFileOverviewChecker(checkers.BaseChecker):
-    """Checks if there is an empty line between file overview and imports."""
+    """Checks if there is an empty line between file overview and imports.
+       Note: The check assumes that all files have a file overview, this
+       assumption is justified because Pylint has an inbuilt check for missing
+       file overview.
```

Comment:
Specify, in parentheses, what the name of this inbuilt check is. Make things as easy for the reader of the code as possible, since you already have the context.

Reply:
done